ui/partials/perf/comparetable.html

```diff
@@ -82,7 +82,10 @@
                     replicates="compareResult.newRuns"></ph-average>
       </td>
       <td ng-class="$ctrl.getCompareClasses(compareResult)">
-        <span ng-if="compareResult.delta" class="detail-hint" uib-tooltip="Mean difference: {{compareResult.delta|displayPrecision}} (= {{compareResult.deltaPercentage|displayPrecision}}%)">
+        <span ng-if="compareResult.delta  && compareResult.newIsBetter" class="detail-hint" uib-tooltip="Mean difference: {{compareResult.delta|displayPrecision}} (= {{compareResult.deltaPercentage|displayPrecision |absoluteValue}}% better)">
```

Comment:
Could you remove the space between "displayPrecision and " " |absoluteValue" here and below?

i.e. `displayPrecision |absoluteValue` -> `displayPrecision|absoluteValue`